### ISSUE TITLE
use cfgexplorer in graph?

### ISSUE THREAD

**Issue by oli-obk** [oli-obk/priroda#7]
@lqd suggested that we have a look at https://github.com/hdc-arizona/cfgexplorer for visualizing loops

**Comment by bjorn3** [comment]
Blocked on hdc-arizona/cfgexplorer#2

Edit: now blocked on new issue
